### ISSUE TITLE
App not launching

### ISSUE THREAD

**Issue by sjdfysdjfysdfjby** [zenangst/KeyboardCowboy#460]
Just downloaded and installed the app via Keyboard.Cowboy.3.19.0.dmg. 

At first, I got the box saying it would request permissions for Accessibility, but the Accessibility settings never popped up. 

Nevertheless I manually went into the Security/Accessibility section and added Keyboard Cowboy, which then launched the app. However there was no GUI window or anything, not sure if there was supposed to be. I could just that see the menu bar options (file, edit, etc) was there. Via this I could go to Settings, but there were only limited options and I'm not sure if that was supposed to the main GUI or not.

Either way, after I closed the settings once and went to another app, I now can't relaunch it again. If I kill KC then I can see the icon bounce in the dock when I relaunch it, but no GUI comes up and I can't see the menu bar either, nothing happens at all. The icon disappears from the dock shortly after, maybe 5 seconds later. Not sure if this is intended behaviour or not.

I've removed and reinstalled more than once, the only thing that changes is that on fresh relaunch I get a request to automatically update. After I click yes, it again disappears and won't respond. I also tried installing 3.18.0, same problem.

On an M1 MBA 16gb running 14.0 (23A344).

**Comment by zenangst** [comment]
Hey @sjdfysdjfysdfjby, that seems a tad off. However, the app is designed to not show itself in the Dock unless the main window is open. When launching the app, can you see it in the menubar?

<img width="251" alt="image" src="https://github.com/zenangst/KeyboardCowboy/assets/57446/ed34a64e-f178-4797-abdb-e3d04081985a">

**Comment by sjdfysdjfysdfjby** [comment]
Ok I think its sorted. Im not sure what exactly fixed it. 

First I found I was able to get to the settings by launching the app and then using Keyboard Maestro to automate going into menu. The menus were there, but not showing in the menubar. Nothing in settings really helped.

I'd gone back to 3.18 so I updated via Check for Updates. I then rebooted, no joy.

I then manually removed the permissions to Accessibility for the app. At first nothing happened, but using KM again to go to the settings, I could see Keyboard Cowboy still thought it had permissions. However after a reboot and relaunch it prompted me again for the permissions for Accessibility, which I agreed to.

It still wouldn't work, but then after another reboot and relaunch I got the prompt to start the app with a blank template or default set. It now runs and I can see the main window if I go to the menubar and choose Open Keyboard Cowboy.

It still doesn't open the main window if I launch the app directly, eg from Alfred. But this might be the intended behaviour? I know some apps can only be invoked from the menu bar.

Anyway I don't know which part of this spaghetti solved the issue, but I'm happy to share any logs etc that might be useful. However it seems to be working for me now, assuming the main window not showing is by design. Thanks!
